libraries/WebServer/examples/Middleware/Middleware.ino

```diff
@@ -0,0 +1,170 @@
+#include <WiFi.h>
```

Comment:
Please add a couple of lines in the beginning of the example explaining how to use the example.